packages/explat-client/src/internal/test/state.ts

```diff
@@ -6,48 +6,37 @@ import { validExperimentAssignment, validFallbackExperimentAssignment } from '..
 
 describe( 'state', () => {
 	it( 'should save and retrieve valid ExperimentAssignments', () => {
-		expect( State.retrieveExperimentAssignment( validExperimentAssignment.experimentName ) ).toBe(
-			undefined
-		);
-		State.storeExperimentAssignment( validExperimentAssignment );
+		const store = State.createStore();
+		expect(
+			State.retrieveExperimentAssignment( store, validExperimentAssignment.experimentName )
```

Comment:
Can we pick a better name than `State`? That name seems rather ambiguous, it may be better to use the name `ExperimentState` or something. I suck at naming things, so take it with a grain of salt.

Reply:
Maybe call it `Store` or `AssignmentStorage`?

Reply:
Going with `ExperimentAssignmentStore` for the class.